src/05-derivatives/01-introduction.md

```diff
@@ -0,0 +1,221 @@
+# Goals and scope
+
+Derivatives are outputs of common processing pipelines, capturing data and
+meta-data sufficient for a researcher to understand and (critically) reuse those
+outputs in subsequent processing. Standardizing derivatives is motivated by use
+cases where formalized machine-readable access to processed data enables higher
+level processing.
+
+## Storage of derived datasets
+
+Derivatives can be stored/distributed in two ways:
+
+1.  Under a `derivatives/` subfolder in the root of the source BIDS dataset
+    folder to make a clear distinction between raw data and results of data
+    processing. Each pipeline has a dedicated directory under which it stores
+    all of its outputs. There are few restriction on the directory name; it is
+    RECOMMENDED to use the format `<pipeline>-<variant>` in cases where it is
+    anticipated that the same pipeline will output more than one variant (e.g.,
+    `AFNI-blurring`, `AFNI-noblurring`, etc.). For the sake of consistency, the
+    subfolder name MUST be a substring of `PipelineDescription.Name` field in
```

Comment:
Can we update this, so it's not a substring but the exact string ? The example below would then be:

```
<dataset>/derivatives/fmriprep-v1/sub-0001
```

Reply:
It is still a substring, but what about:

```
the `PipelineDescription.Name` field MUST be a prefix of the subfolder name,
optionally followed by a hyphen and a suffix.
```

Or similar?

Reply:
Ah, yes, sorry, it's a substring of the whole string ! But the exact string before the hyphen :) 

What about :

```
the subfolder name MUST be the `PipelineDescription.Name` field in `data_description.json`,
optionally followed by a hyphen and a suffix.
```

Reply:
Applied your suggestion, @emdupre.

Reply:
Thank you !